### DISCUSSION TITLE
Multiple source repo under applicationsets having issues

### DISCUSSION THREAD

**Discussion by gayathrithathinol** [argoproj/argo-helm#2784] [General]
We are trying to use multiple sources in applicationsets by defining the "ref". However we are getting error on the same and values are not getting referenced by this .

need help on this feature.

sources:
        - repoURL: https://*****************secondaryvalues.git
          targetRevision: master
          path: values-kms-qa
          ref: kms-qa
        - repoURL: https://*****************primaryapp.git
          path: ccid-kms
          targetRevision: master-k8s
          helm: 
            releaseName: 'ccid-kms'
            valueFiles: 
              - $kms-qa/values.yaml
              - $kms-qa/values-secrets.yaml

**Comment by yu-croco** [comment]
Hi @gayathrithathinol , since it's about Argo CD's specific feature, you can get more help in upstream.
